Makefile

```diff
@@ -47,10 +47,11 @@ dist/docker/mesos-calico.tar: docker_image
 
 jenkins: calico_mesos
 	docker build -t calico/mesos-calico .
-
-jenkins-vagrant: calico_mesos docker_image.tar
-	vagrant up
-
+	export MESOS_CALICO_TAR=true
+	docker build -t calico/mesos-calico .
+	mkdir -p dist/docker
```

Comment:
We probably want to have a call to `make clean` on the jenkins server before we run `make jenkins` to start fresh each time.